my-app/src/services/employees.service.js

```diff
@@ -0,0 +1,62 @@
+import Constants from "../constants/constants.service";
+import authService from "./auth.service";
+import departmentsService from "./departments.service";
+
+const baseUrl = Constants.getBaseUrl();
+
+const getEmployees = () => {
+  const options = {
+    method: "GET",
+    headers: {
+      authorization: authService.getToken(),
+    },
+  };
+
+  return fetch(`${baseUrl}/employees`, options).then(async (res) => {
+    if (!res.ok) {
+      throw new Error(res.statusText);
+    }
+
+    const data = await res.json();
+    return data?.employees || [];
+  });
+};
+
+const getEmployeesByDepartmentId = (departmentId) => {
+  const options = {
+    method: "GET",
+    headers: {
+      authorization: authService.getToken(),
+    },
+  };
+
+  return fetch(
+    `${baseUrl}/departments/${departmentId}/employees`,
+    options
+  ).then(async (res) => {
+    if (!res.ok) {
+      throw new Error(res.statusText);
+    }
+
+    const data = await res.json();
```

Comment:
The same is here

Reply:
I agree with the idea of moving the object outside of the function, but the variable is smaller. If I put it outside, the code won't become more readable (as in the case with the object), but I will create one more line of code. Maybe I don't quite understand...

Reply:
you create this code 
```suggestion
    options = {
    method: "GET",
    headers: {
      authorization: authService.getToken(),
    },
```
on line 8 and 26, its unnecessary duplication

Reply:
aaahhh, omg, sorry I didn't get it)) Now I see!!! Thanks!!!